std/complex.d

```diff
@@ -388,6 +388,7 @@ struct Complex(T)  if (isFloatingPoint!T)
     ref Complex opOpAssign(string op, R)(R r)
         if (op == "^^" && isFloatingPoint!R)
     {
+        static import std.math;
```

Comment:
ditto... even though arguably it's an annoyance to have to type out all the selected symbols for each import line.  I dunno what's the best solution... a straight out `import std.math` runs the risk of (unintentional) hijacking.  But still, I'm leaning towards keeping the mathematical expressions less verbose.